CMake/AbseilHelpers.cmake

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright 2017 The Abseil Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include(CMakeParseArguments)
+
+set(_ABSEIL_HELPERS_PATH "${CMAKE_CURRENT_LIST_DIR}")
+
+#
+# create a static library based on the following variable
```

Comment:
Why static?
Anything wrong with letting the CMake user choose the type with BUILD_SHARED_LIBS?

Reply:
Also note that, on Windows .dll are runtime artifacts so RUNTIME DESTINATION should be used in this case.

Reply:
First comment: after discusssions with Google guys, we agree to export only static for now. 

Second comment: fixed.